codegen/src/test/java/org/web3j/codegen/unit/gen/MethodSpecGeneratorTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.codegen.unit.gen;
+
+import org.junit.jupiter.api.Test;
+
+public class MethodSpecGeneratorTest {
+
+    @Test
+    public void test() {
+        // GenericTemplate genericTemplate = new GenericTemplate("Test");
```

Comment:
Can we fix this? We shouldn't have commented out code.

Reply:
Fixed